luigi/lock.py

```diff
@@ -50,6 +50,8 @@ def getpcmd(pid):
                 spid, scmd = line.strip().split(' ', 1)
                 if int(spid) == int(pid):
                     return scmd
+    # Fallback instead of None, for e.g. Cygwin where -o is an "unknown option" for the ps command:
```

Comment:
is there a better way to detect this rather than just always assuming it's cygwin if everything else fails?

i think you can check if `sys.platform == 'cygwin'`